core/controllers/base.py

```diff
@@ -411,6 +411,16 @@ def vaidate_and_normalize_args(self):
             arg: normalized_value.get(arg) for arg in request_arg_keys
         }
 
+        default_value_keys = list(
```

Comment:
I don't know what this means, conceptually. I don't see any reference to default values here in the logic.

Reply:
Added comment PTAL, Thank you.